### PR TITLE
NCL 766 : Update versions during build

### PR DESCRIPTION
**Pull request by rnc**
@alexcreasy  @vibe13  : can you review please

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1230,7 +1278,57 @@
...
+
+                                <property name="newversion" value="${project.rel.org.jboss.pnc:parent}"/>
+
+                                <replaceregexp file="${basedir}/bower.json"
+                                               match="version&quot;: &quot;[0-9].*&quot;"
+                                               replace="version&quot;: &quot;${newversion}&quot;"/>
+                                <replaceregexp file="${basedir}/package.json"
+                                               match="version&quot;: &quot;[0-9].*&quot;"
+                                               replace="version&quot;: &quot;${newversion}&quot;"/>
+                                <replaceregexp file="${basedir}/npm-shrinkwrap.json"
```
Comment (vibe13) [inline pom.xml]: @rnc These replacements (L1316-1324) scare me a bit, there are tons of versions inside those files, and versions of resolved dependencies could potentially match the PNC version. But seems that only npm-shrinkwrap.json file has many  "version": "X.Y.Z" declarations, so package.json and bower.json should be fine. We could remove the npm-shrinkwrap.json file from the replacement or enforce a more strict replacement to be sure that only the pnc-ui version gets replaced.

Comment (vibe13) [inline pom.xml]: All the remaining bits looks good!

**Comment by vibe13** [comment]
Looks very good now! +1 also for the JBOSS_HOME env variable check!

**Comment by alexcreasy** [comment]
Yes this looks good to me.

**Pull request merged**